index.html

```diff
@@ -186,54 +187,64 @@ <h3 id="tier-enterprise" class="text-base/7 font-semibold text-indigo-400">Pack
+                <input class="form-check-input" type="checkbox" value="Branding|50" id="flexCheckDefault00">
+                <label class="form-check-label" for="flexCheckDefault00">Branding</label>
+            </div>
+            <div class="form-check">
+                <input class="form-check-input" type="checkbox" value="Edición de video|50" id="flexCheckChecked01" checked>
+                <label class="form-check-label" for="flexCheckChecked01">Edición de video</label>
+            </div>
+            <div class="form-check">
+                <input class="form-check-input" type="checkbox" value="Material Publicitario|50" id="flexCheckChecked02" checked>
+                <label class="form-check-label" for="flexCheckChecked02">Material Publicitario</label>
+            </div>
+            <div class="form-check">
+                <input class="form-check-input" type="checkbox" value="Diseño de Packaging|50" id="flexCheckChecked03" checked>
+                <label class="form-check-label" for="flexCheckChecked03">Diseño de Packaging</label>
+            </div>
+            <div class="form-check">
+                <input class="form-check-input" type="checkbox" value="Diseño Web|50" id="flexCheckChecked04" checked>
+                <label class="form-check-label" for="flexCheckChecked04">Diseño Web</label>
+            </div>
+            <div class="form-check">
+                <input class="form-check-input" type="checkbox" value="Diseño Editorial|50" id="flexCheckChecked05" checked>
+                <label class="form-check-label" for="flexCheckChecked05">Diseño Editorial</label>
+            </div>
+            <div class="form-check">
+                <input class="form-check-input" type="checkbox" value="Diseño UX|50" id="flexCheckChecked06" checked>
+                <label class="form-check-label" for="flexCheckChecked06">Diseño UX</label>
+            </div>
+            <div class="form-check">
+                <input class="form-check-input" type="checkbox" value="Email Design|50" id="flexCheckChecked07" checked>
+                <label class="form-check-label" for="flexCheckChecked07">Email Design</label>
+            </div>
+            <div class="form-check">
+                <input class="form-check-input" type="checkbox" value="Animación|50" id="flexCheckChecked08" checked>
+                <label class="form-check-label" for="flexCheckChecked08">Animación</label>
+            </div>
         </div>
         <button id="botonArmarPack" class="boton-armar-pack" onclick="armarPack()">Armar Pack</button>
-        <!-- <button id="botonArmarPack" class="boton-armar-pack">Armar Pack</button> -->
       </section>
 
+      <section id="resultadoPack" class="resultado-pack">
+        <h2>Carrito</h2>
+        <div id="carrito"></div>
+        <p id="precioTotal">Precio Total: $0</p>
+        <button id="finalizarCompra" class="btn-finalizar" onclick="finalizarCompra()">Finalizar Compra</button>
```

Comment:
Esto esta mal, no asociamos asi los eventos, lo hacemos directamente en js, no en el HTML incrustado:
```suggestion
document.getElementById("btnFinalizarCompra").addEventListener("click", finalizarCompra);

```